deploy/crds/workspace.che.eclipse.org_workspaces_crd.yaml

```diff
@@ -5,24 +5,24 @@ metadata:
 spec:
   additionalPrinterColumns:
   - JSONPath: .status.workspaceId
-    name: Id
+    description: The workspace's unique id
+    name: Workspace ID
```

Comment:
it's not related to this PR but still interesting topic to raise:
Does Workspace ID makes sense since workspace is CR?
Do you think we can reuse UID?

Sometime ago I tried to remove it, but workspace id now it's workspace word + cut version of UID, which is used for objects/names generation. But I wonder if it makes sense in general and if we can drop it out.

Reply:
One reason for leaving it in is that we use workspace ID (which is a just the object's UID, but modified) in various places and as a label for all created objects. We *could* use UID directly, but parsing the UID to a UUID [can error](https://github.com/amisevsk/che-workspace-crd-operator/blob/93fdedbd3983cc501aab36a40f16f412f0b888d6/pkg/controller/workspace/workspace_controller.go#L281-L287), so it would complicate using a unique ID the way we do. We would also still have to store it to pass to subcomponents, since their UID is different from the main workspace UID.

Another option would be using the CR's name as a base, so everything created for `cloud-shell` would be prefixed by `cloud-shell`.